shopping-server/views/layout.jade

```diff
@@ -2,6 +2,6 @@ doctype html
 html
   head
     title= title
-    link(rel='stylesheet', href='/stylesheets/style.css')
+    link(rel='stylesheet', href='../public/stylesheets/style.css')
```

Comment:
You do not need to do this. It is automatically connected with https://github.com/Viveckh/Veniqa/blob/develop/shopping-server/app.js#L87

Reply:
Ok I got it. Thanks:)